docs/configuration.md

```diff
@@ -47,5 +46,6 @@ The table below enumerates each individual setting, it's expected type, it's def
 | data_file_search_directories | list | ["./"] The paths at which to start searching for files with structured data in them to validate against defined schemas. This path is relative to the directory in which `schema-enforcer` is executed.
```

Comment:
Looks like perhaps `validator_directory` needs to be added to this table and to the toml above?